postgres/defaults.yaml

```diff
@@ -47,6 +47,12 @@ postgres:
     - ['host', 'all', 'all', '127.0.0.1/32', 'md5']
     # IPv6 local connections:
     - ['host', 'all', 'all', '::1/128', 'md5']
+  acls_v10:
```

Comment:
Syntax and content looks fine. Consider changing identifier name from `acls_v10` to `acls_replication`.

Reply:
I'm not sure that having multiple settings for ACL is really that good idea.
I have checked PostgreSQL docs, and it seams the replication settings are supported starting at lease from PG [release 9.0](https://www.postgresql.org/docs/9.0/static/auth-pg-hba-conf.html).

So I think it would be no harm to include these ACL rules by default for any PG version.